integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanTransactionReverseReplayTest.java

```diff
@@ -360,6 +361,81 @@ public void loanTransactionReverseReplayWithChargeOffAndCBR() {
         }
     }
 
+    @Test
+    public void loanTransactionReverseReplayWithChargeOff() {
```

Comment:
Can we create the test (probably in a separate test class file) using the BaseLoanIntegrationTest? Looks much cleaner and easier to maintain.